sktime/split/expandinggreedy.py

```diff
@@ -39,7 +44,7 @@ class ExpandingGreedySplitter(BaseSplitter):
     Examples
     --------
     >>> import numpy as np
-    >>> from sktime.split import ExpandingGreedySplitter
+    >>> from sktime.forecasting.model_selection import ExpandingGreedySplitter
```

Comment:
A bit confused - I thought we are moving to `sktime.split`? Does that import not work? I see it's exported in `sktime/split/__init__.py` module of `expandinggreedysplitter-float` branch as well.

Reply:
accidental revert

Reply:
fixed